build-system/tasks/runtime-test/status-report.js

```diff
@@ -30,6 +30,17 @@ const IS_SAUCELABS = !!(argv.saucelabs || argv.saucelabs_lite);
 const IS_SINGLE_PASS = !!argv.single_pass;
 const IS_UNIT = !!argv.unit;
 
+const TEST_TYPE_SUBTYPES = new Map([
+  // TODO(danielrozenberg): add 'saucelabs' to integration tests when supported.
+  ['integration', ['default', 'single-pass']],
```

Comment:
From initial feedback, `default` was confusing. Let's rename it to `local`, since that's what it's called elsewhere.

/cc @estherkim

Reply:
Renamed to `local`. Esther, feel free to suggest a different name if you don't like this one. I can change it in a followup PR.

Reply:
End of 🚲 🏠